lib/hipmer/hipmerUtils.py

```diff
@@ -448,12 +470,27 @@ def finish_run(self, params):
             raise RuntimeError("Error in HipMER execution")
 
         wsname = params['workspace_name']
-        self.log(console, 'Uploading FASTA file to Assembly')
+
+        self.log(console, 'Filtering short length contigs from HipMer assembly')
+
         assemblyUtil = AssemblyUtil(self.callbackURL, token=self.token)
+
+        assembly_size_filter = params['assembly_size_filter']
+
+        filtered_fasta_file_path = self.filter_contigs_by_length(output_contigs, assembly_size_filter)
+
+        if os.stat(filtered_fasta_file_path).st_size == 0:
+            raise ValueError("Error: Using input parameters, you have filtered all contigs from the HipMer assembly. Increase the minimum contig size and try again.")
```

Comment:
Think this should be decrease not increase?

Reply:
also line wrap

Reply:
thanks, good catches